broker/aws/impl.go

```diff
@@ -203,11 +203,16 @@ func (b *Broker) getAWSRolesForAccount(accountName string) ([]string, error) {
 }
 
 func stringIntersectionNoDups(set1, set2 []string) (intersection []string) {
+	var stringMap map[string]string
+	stringMap = make(map[string]string)
```

Comment:
This compiles?

Reply:
Please replace both lines with one: `stringMap := make(map[string]string)`

Reply:
Actually, this would be optimal: `stringMap := make(map[string]string, len(v1))`